jdbc/src/main/java/org/apache/metamodel/jdbc/JdbcDropTableBuilder.java

```diff
@@ -63,10 +64,15 @@ public void execute() {
     }
 
     protected String createSqlStatement() {
-        FromItem fromItem = new FromItem(getTable());
-        String tableLabel = _queryRewriter.rewriteFromItem(fromItem);
+        final Table table = getTable();
+        final FromItem fromItem = new FromItem(table);
+        final String tableLabel = _queryRewriter.rewriteFromItem(fromItem);
 
-        return "DROP TABLE " + tableLabel;
+        if (table.getType() != null && table.getType() == TableType.VIEW) {
+            return "DROP VIEW " + tableLabel;
```

Comment:
I wonder if we could simply go like this:

```
TableType type = table.getType() == null ? TableType.TABLE : table.getType();
return "DROP " + type + " " + tableLabel;
```

Oh, and I would maybe rename `tableLabel` to `qualifiedTableName` or something.

Reply:
TableType has the following values: TABLE, VIEW, SYSTEM_TABLE, GLOBAL_TEMPORARY, LOCAL_TEMPORARY, ALIAS, SYNONYM, OTHER.
In SQL you either drop a table or a view (you don't drop a system_table, that's a table). I would say that the VIEW value is only relevant for droping views. All the other values beloing to droping tables.
TableType has no toString implementation. You could implement a custom toString just for this JdbcDropTableBuilder#createSqlStatement() method, but I don't think that would be a very representable implementation of that method.

Reply:
I'll change the variable name.